dp_branding/index.php

```diff
@@ -36,121 +36,202 @@
-        <td class='gray'>
-            <img src='dp-logo-500px.png'><br>
-            <a href='dp-logo-500px.png'>dp-logo-500px.png</a>
-        </td>
-    </tr>
-    <tr>
-        <th>White<br>Background</th>
-        <td>
-            <img src='dp-logo-500px-white.png'><br>
-            <a href='dp-logo-500px-white.png'>dp-logo-500px-white.png</a>
-        </td>
-    </tr>
-    </table>
+        <table class='branding'>
+        <tr>
+            <td>Transparent<br>Background, light themes</td>
+            <td style='background-color:white'>
+                <img src='dp-logo-500px.png'>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <a href='dp-logo-500px.png'>dp-logo-500px.png</a>
+            </td>
+        </tr>
+        <tr>
+            <td>White<br>Background</td>
+            <td>
+                <img src='dp-logo-500px-white.png'>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <a href='dp-logo-500px-white.png'>dp-logo-500px-white.png</a>
+            </td>
+        </tr>
+        </table>
     LOGO;
 
-echo "<h3 id='bw_logo'>" . _("Black &amp; White Logo") . "</h3>";
-
-echo "<p><b>" . _("SVG file") . ":</b> <a href='dp-logo-bw.svg'>dp-logo-bw.svg</a></p>";
-
-echo "<p><b>" . _("PNG files") . ":</b></p>";
+echo "<h3 id='bw_logo'>" . _("Solid Black Logo") . "</h3>";
+
+echo "<h4>" . _("SVG file") . "</h4>";
+
+echo <<<SVGLOGO
+        <table class='branding'>
+        <tr>
+            <td>Transparent<br>Background</td>
+            <td class='black-transparent'>
+                <img src='dp-logo-bw.svg'>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <a href='dp-logo-bw.svg'>dp-logo-bw.svg</a>
+            </td>
+        </tr>
+        </table>
+    SVGLOGO;
+
+echo "<h4>" . _("PNG file") . "</h4>";
 
 echo <<<LOGO
-    <table class='branding'>
-    <tr>
-        <th class='black-transparent'>Transparent<br>Background</th>
-        <td class='black-transparent'>
-            <img src='dp-logo-bw-500px.png'><br>
-            <a href='dp-logo-bw-500px.png'>dp-logo-bw-500px.png</a>
-        </td>
-    </tr>
-    </table>
+        <table class='branding'>
+        <tr>
+            <td>Transparent<br>Background</td>
+            <td class='black-transparent'>
+                <img src='dp-logo-bw-500px.png'>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
+            <td>
+                <a href='dp-logo-bw-500px.png'>dp-logo-bw-500px.png</a>
+            </td>
+        </tr>
+        </table>
     LOGO;
 
 echo "<h3 id='mark'>" . _("Mark") . "</h3>";
 
-echo "<p><b>" . _("SVG file") . ":</b> <a href='dp-mark.svg'>dp-mark.svg</a></p>";
-
-echo "<p><b>" . _("PNG files") . ":</b></p>";
+echo "<h4 id='mark'>" . _("SVG file") . "</h4>";
+
+echo <<<SVGMARK
+        <table class='branding'>
```

Comment:
Thought: should we make this SVG table like the Logo tables above where the left column has the "Transparent background" label and the right is the mark with the link?

Reply:
Good idea. I've got a meeting coming up shortly, but I'll do that afterwards.

Reply:
Done.